functions/whois/whois_qrz_telegram.py

```diff
@@ -0,0 +1,49 @@
+from functions.default import *
+from telegram import InlineKeyboardMarkup, InlineKeyboardButton, Update
+from telegram.ext import CallbackQueryHandler, CommandHandler, ConversationHandler
+from telegram.constants import ParseMode
+from qrz import QRZ
+
+def print_keys(key_names, query_result):
+    info = ""
+    for key_name in key_names:
+        if key_name in query_result:
+            info += query_result[key_name] + " "
+    print(info)
+    return info
+
+async def whois_qrz_query(update: Update, context: ContextTypes.DEFAULT_TYPE) -> int:
+    log_func("whois_qrz_query()", update)
+    username = update.message.from_user["username"]
+
+    if check_permissions(username, update, context):
+        if len(context.args) > 0:
+            callsign = context.args[0].upper()
+
+            await context.bot.send_message(
+                chat_id=update.effective_chat.id,

```

Comment:
Not necessary to write to chat.

See camera example:

await context.bot.send_chat_action(
        chat_id=update.effective_chat.id, action=ChatAction.TYPING
    )